Makefile

```diff
@@ -27,7 +27,7 @@ endif
 
 APPNAME      = "TON"
 APPVERSION_M = 2
-APPVERSION_N = 4
+APPVERSION_N = 6
 APPVERSION_P = 1
```

Comment:
0

Reply:
Yes indeed, but 2.6.1 has already been deployed on P3, next release will be `2.6.2`